src/main/java/seedu/address/model/util/Name.java

```diff
@@ -18,7 +18,7 @@
      */
     public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
 
-    public final String fullName;
+    public final String name;
```

Comment:
This property should be called `value`
`getName().name` sounds a bit unsemantic as the property name has the same name as class name (haha so many names)

Reply:
Updated in code.